src/sr_od/app/sim_uni/sim_uni_app.py

```
@@ -51,6 +58,24 @@ def __init__(self, ctx: SrContext,
         self.not_found_in_survival_times: int = 0  # 在生存索引中找不到模拟宇宙的次数
         self.all_finished: bool = False
 
+    # 在差分宇宙入口处检查周期奖励
+    def _check_period_reward(self) -> OperationRoundResult:
+        ocr_result_map = self.ocr(self.ctx.controller.screenshot(), '模拟宇宙', '差分宇宙-每周奖励')
+        # 找到了2个 14000 的算过 (完成进度 14000/14000)
+        # 找不到 14000 的也算过 (防止死循环)
+        count_14000 = 0
+        for ocr_result, mrl in ocr_result_map.items():
```

Ruff (0.13.1)

Loop control variable `mrl` not used within loop body

Rename unused `mrl` to `_mrl`

(B007)